web/templates/note/index.html.eex

```diff
@@ -1,9 +1,11 @@
 <h1>Listing Notes</h1>
 
+<p><%= link "New note", to: note_path(@conn, :new) %></p>
+
 <ul>
   <%= for note <- @notes do %>
     <li>
-      <div><b><%= note.title %></b></div>
+    <div><b><%= link note.title, to: note_path(@conn, :show, note.id) %></b> (<%= note.id %>)</div>
```

Comment:
That's fine, although, you can also hover over it. You can see the link in the url that chrome displays on the bottom left. Up to you.